README-dev-on-docker.md

```diff
@@ -0,0 +1,114 @@
+# Developing Vespa on Docker
+
+The purpose of this document is to describe how to set up a combined development and
+run-time environment for (iteratively) developing and testing Vespa.
+
+### Start the docker container
+
+    DOCKER_IMAGE=vespaengine/vespa-dev:latest
+    SOURCE_ROOT=${HOME}/vespa/vespa
```

Comment:
I prefer 'git' (or 'github', as other guides use) instead of the first 'vespa' (same for line below)